### ISSUE TITLE
Implementing a Note Application -

### ISSUE THREAD

**Issue by ggirton** [survivejs/react#414]
" If you try debugging it, you can see the ids should change if you refresh. You can verify this easily either by inserting a console.log(notes); line or a debugger; statement within the component function."

I did not find a place where inserting console.log(notes); did not cause the page to fail to display.  An example here would help.

**Comment by bebraw** [comment]
Yeah, that's a valid point. I'll try adding more context there so it's easier to get the idea. Thanks!

**Comment by ggirton** [comment]
I was able to get logging to work after A) encapsulating entire return value in a <div> and B) looking for it in the browser console (as opposed to the terminal console).   Many thanks for this fine introduction … I will have a couple more minor suggestions as I g along, based on my unfamiliarities with the toolsets :-)  CHeers!

On Jun 21, 2016, at 9:37 PM, Juho Vepsäläinen <notifications@github.com> wrote:

Yeah, that's a valid point. I'll try adding more context there so it's easier to get the idea. Thanks!

**Comment by bebraw** [comment]
Cool. Keep them coming. 👍
